.github/workflows/spellchecking.yml

```diff
@@ -0,0 +1,16 @@
+name: spellchecking
+
+on: [pull_request]
+
+jobs:
+  spellchecking:
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@master
+        name: Check out the code
+      - uses: actions/setup-node@master
+        name: Run spell check
+        with:
+          node-version: "18"
+      - run: npm install -g cspell
+      - run: cspell --config ./cSpell.json "**/*.md"
```

Comment:
If we configure this workflow as required for all repos in the organisation (https://docs.github.com/en/actions/using-workflows/required-workflows), will it be necessary for all the repos to contain the `cSpell.json` file? Can we make it included in the workflow?

Reply:
At first, we only intend to use it on the .md to see how it behaves